.travis.yml

```diff
@@ -28,10 +28,10 @@ install:
 - npm install
 - docker build -t cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER .
 - if [ "$TRAVIS_BRANCH" == "release" ]; then
-    docker build -t cloudboost/cloudboost:latest .;
+    docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:latest;
   fi
 - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker build -t cloudboost/cloudboost:staging .;
+    docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER-master;
```

Comment:
Just push to :master and do not have versioning in this case.

Reply:
Oh okay, so just `cloudboost/cloudboost:master` then.

Reply:
Yes.